src/DatePickerRange/DatePickerRange.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { DatePickerRange } from '../DatePickerRange';
+import { boolean, select } from '@storybook/addon-knobs/react';
+
+storiesOf('DatePickerRange', module).add('DatePickerRange', () => (
+  <DatePickerRange
+    showDefaultInputIcon={boolean('showDefaultInputIcon', true)}
+    inputIconPosition={select(
+      'inputIconPosition',
+      ['before', 'after'],
+      'before',
+    )}
+    small={boolean('isSmall', true)}
```

Comment:
Let's match the prop name with the knob label

Reply:
Aha. Ok.